jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java

```diff
@@ -1155,10 +1155,14 @@ protected HttpField getAcceptEncodingField()
         return encodingField;
     }
 
+    /**
+     * @param host the host to normalize
+     * @return the host itself
+     * @deprecated no replacement, do not use it
+     */
+    @Deprecated
```

Comment:
should the impl at least call HostPort.normalize so that it works if somebody does call it?

Reply:
Also, perhaps add some apidoc text telling developers what the options are now that it's deprecated ...

``` java
/**
 * @deprecated use {@link HostPort#normalize(String)} instead.
 */
@Deprecated
```

Reply:
Nope, the method should not be used. It's `protected` so only subclasses could use it, but they should not - there is no need to normalize the host anymore.

I initially followed @gregw suggestion to call `HostPort.normalizeHost()`, but actually this method was doing exactly the opposite. So either restore the old behavior, or keep this new one that does nothing.